src/appendices/phenotype.md

```diff
@@ -0,0 +1,326 @@
+    ses-MRI2/
+        anat/
+            sub-01_ses-MRI2_T1w.json
+            sub-01_ses-MRI2_T1w.nii.gz
+sub-02/
+    ses-MRI1/
+        anat/
+            sub-02_ses-MRI1_T1w.json
+            sub-02_ses-MRI1_T1w.nii.gz
+```
+
+Contents of `phenotype/<measurement_tool_name>.tsv`
+
+```Text
+participant_id session_id measurement_1 measurement_2
+sub-01         ses-pheno1 value1        value2
+sub-02         ses-pheno1 value3        value4
+sub-02         ses-pheno2 value5        value6
+```
+
+### 3 participants with 3 different kinds of sessions among them
+
+The `ses-baseline` session collects an MRI and tabular phenotypic data.
+
+File tree
+
+```Text
+participants.json
+participants.tsv
+sessions.json
+sessions.tsv
+phenotype/
+    demographics.json
+    demographics.tsv
+    ...
+sub-01/
+    ses-baseline/
+    ses-followupMRI/
+sub-02/
+    ses-baseline/
+sub-03/
+    ses-baseline/
+    ses-followupMRI/
+```
+
+Contents of `sessions.tsv`.
+
+```Text
+participant_id session_id      acq_time            age
+sub-01         ses-baseline    2001-01-01T12:05:00 10
+sub-01         ses-followupMRI 2001-07-01T13:33:00 10
+sub-01         ses-interview   2002-01-01T11:21:00 11
+sub-02         ses-baseline    2001-04-01T11:01:00 9
+sub-02         ses-interview   2002-04-01T14:08:00 10
+sub-03         ses-baseline    2001-09-01T11:45:00 11
+sub-03         ses-followupMRI 2002-03-01T12:17:00 12
+```
+
+Contents of `sessions.json`. Note how the `session_id` `Levels` are clearly described.
+
+```json
+{
+    "participant_id": {
+        "Description": "BIDS participant identifier"
+    },
+    "session_id": {
+        "Description": "BIDS session identifier",
+        "Levels": {
+            "ses-baseline": "Baseline visit for MRI and assessments",
+            "ses-followupMRI": "6-months after baseline MRI follow-up",
+            "ses-interview": "1-year after baseline in-person follow-up"
+        }
+    },
+    "acq_time": {
+        "Description": "When the data acquisition started"
+    },
+    "age": {
+        "Description": "Age of the participant at the session",
+        "Units": "years"
+    }
+}
+```
+
+Contents of `participants.tsv`. Note how this file only contains age at
+the earliest session. This is intended for the expectation of 1 row per participant
+in the participant file.
+
+```Text
+participant_id age sex
+sub-01         10  M
+sub-02         9   F
+sub-03         11  F
+```
+
+Contents of `phenotype/demographics.tsv`. Measures or features that can change
+from session to session belong here especially.
+
+```Text
+participant_id session_id      gender race household_income
+sub-01         ses-baseline    3      4    5
+sub-01         ses-followupMRI 3      4    5
+sub-01         ses-interview   4      4    6
+sub-02         ses-baseline    1      3    3
+sub-02         ses-interview   1      7    3
+sub-03         ses-baseline    2      10   4
+sub-03         ses-followupMRI 5      10   4
+```
```

Comment:
Is this aligned with our latest discussion? 
1. Age at the earliest session in `participants.tsv` vs `demographics.tsv` ?
2. Age in sessions.tsv while we already have changing variables in `demographics.tsv` ?